examples/curl.rs

```diff
@@ -4,9 +4,9 @@ use ngx::ffi::{
     ngx_uint_t, NGX_CONF_TAKE1, NGX_HTTP_LOC_CONF, NGX_HTTP_MODULE, NGX_RS_HTTP_LOC_CONF_OFFSET,
     NGX_RS_MODULE_SIGNATURE,
 };
-use ngx::http::MergeConfigError;
-use ngx::{core, core::Status, http, http::HTTPModule};
-use ngx::{http_request_handler, ngx_log_debug_http, ngx_modules, ngx_null_command, ngx_string};
+use ngx::{core, core::Status, http, http::HTTPModule, http::MergeConfigError};
+use ngx::{http_request_handler, ngx_log_debug, ngx_modules, ngx_null_command, ngx_string};
```

Comment:
This is really just stylistic, but why not all in one:

`use ngx::{core, core::Status, http, http::{HTTPModule, MergeConfigError}, http_request_handler, ngx_log_debug, ngx_modules, ngx_null_command, ngx_string};`